src/ai/backend/storage/netapp/__init__.py

```diff
@@ -117,6 +125,122 @@ async def create_vfolder(
             lambda: vfpath.mkdir(0o755, parents=True, exist_ok=False),
         )
 
+    async def delete_vfolder(self, vfid: UUID) -> None:
+        vfpath = self.mangle_vfpath(vfid)
+
+        # extract target_dir from vfpath
+        target_dir = str(vfpath).split(self.netapp_qtree_name + "/", 1)[1].split("/")[0]
+        nfs_path = (
+            f"{self.netapp_xcp_hostname}:/{self.netapp_volume_name}/"
+            + f"{self.netapp_qtree_name}/{target_dir}"
+        )
+
+        async def watch_delete_dir(root_dir):
+            # remove vfolder by xcp command
+            proc = await asyncio.create_subprocess_exec(
+                *[
+                    "docker",
+                    "exec",
+                    self.netapp_xcp_container_name,
+                    "xcp",
```

Comment:
Let's merge this 4 lines into a single option, so that users may specify either just `["xcp"]` or `["docker", "exec", "<container-name>", "xcp"]` to support official CentOS setup.